packages/itwinui-css/src/table/table.scss

```diff
@@ -320,17 +325,19 @@
   // Allows columns with static width to take more space than the table width
   align-items: flex-start;
   @include iui-scroll-snapping('.iui-table-row');
-  background-color: var(--iui-color-background-1);
+  background-color: var(--iui-color-background);
 
   &:last-child {
     border-bottom-left-radius: var(--iui-border-radius-1);
     border-bottom-right-radius: var(--iui-border-radius-1);
   }
 
   &.iui-zebra-striping .iui-table-row:nth-child(even):not([aria-selected='true']) .iui-table-cell {
-    @include pseudo-content {
-      background-color: hsl(var(--iui-color-foreground-1-hsl) / 0.02);
-    }
+    background-color: var(--iui-color-background-zebra);
+  }
+
+  &.iui-zebra-striping .iui-table-row:nth-child(even):not([aria-selected='true']):hover .iui-table-cell {
+    background-color: var(--iui-color-background-hover);
```

Comment:
this is already set in the base row hover styles. i think you just need to reduce the specificity of the zebra striping selector above using `:where`

Reply:
We tried to simplify this but decided to leave it as is, table needs a more thorough refactor.